TypeCobol.LanguageServer/VsCodeProtocol/Text Document Notifications/DidChangeTextDocumentParams.cs

```diff
@@ -8,10 +8,21 @@ namespace TypeCobol.LanguageServer.VsCodeProtocol
     /// <summary>
     /// The change text document notification's parameters.
     /// </summary>
-    public class DidChangeTextDocumentParams : TextDocumentIdentifier
+    public class DidChangeTextDocumentParams
     {
+        public VersionedTextDocumentIdentifier textDocument { get; set; }
+
         public TextDocumentContentChangeEvent[] contentChanges { get; set; }
 
-        public DidChangeTextDocumentParams(string uri) : base(uri) { }
+        /// <summary>
+        /// Constructor
+        /// </summary>
+        /// <param name="textDocument">The document that did change.</param>
+        /// <param name="contentChanges">The actual content changes.</param>
+        public DidChangeTextDocumentParams(VersionedTextDocumentIdentifier textDocument, TextDocumentContentChangeEvent[] contentChanges)
```

Comment:
For protocol objects I prefer not to define any constructor, this ensure the deserializer will always find a way to create an instance. Use object initializer to create and define properties in one go.

Reply:
Done